industrial_trajectory_filters/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(industrial_trajectory_filters)
 
+add_compile_options(-std=c++11)
```

Comment:
What's the status of this pull request? If there are many ways to skin this cat, but the difference is minute, then perhaps we could merge this and iterate on the implementation as we go.